packages/app/src/flame/Flam3.tsx

```diff
@@ -80,9 +80,12 @@ export function Flam3(props: Flam3Props) {
     }
   }
 
-  const factor = createMemo(
-    () => (camera.zoom() * pixelRatio()) ** 2 / (props.pointCount / 1e5),
-  )
+  const factor = createMemo(() => {
```

Comment:
Is this exposure factor, maybe we can call it that?

Reply:
oh yeah, I'll rename quickly

Reply:
well... not exactly, it's more of a count adjustment factor, which keeps the bucket counts similar regardless of zoom level, point counts or resolution

Reply:
exposure is applied after the `pow` curve and this factor is applied before